src/jsonTypings.ts

```diff
@@ -267,4 +267,39 @@ export namespace JSONObjects {
     export type JSON = Record<string, Random | Obstacle | Wall | Building | Structure | LootTier | Decal>;
 }
 
+export namespace JSONConfig {
+
+    interface ServerConfig {
+        host: string
+        port: number
+        https: boolean
+        webSocketHost: string
+        webSocketPort: number
+        webSocketHttps: boolean
+        webSocketDevAddress: string
+        useWebSocketDevAddress: boolean
+        webSocketRegions: {
```

Comment:
`Config.webSocketRegions` should be typed as `Record<string, string>` as the keys are not explicitly used by their names (i.e. `Config.webSocketRegions.na`) in the codebase.

Of course, `Config.defaultRegion` must be a valid region from `Config.webSocketRegions`, but that cannot be type-checked at compile time.